ReactWindows/ReactNative/Views/WebView/ReactWebViewManager.cs

```diff
@@ -0,0 +1,289 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Controls;
+using ReactNative.Views.WebView.Events;
+using Windows.Web.Http;
+
+namespace ReactNative.Views.WebView
+{
+    /// <summary>
+    /// A view manager responsible for rendering webview.
+    /// </summary>
+    public class ReactWebViewManager : SimpleViewManager<Windows.UI.Xaml.Controls.WebView>
+    {
+        private const string BLANK_URL = "about:blank";
+
+        private const int CommandGoBack = 1;
+        private const int CommandGoForward = 2;
+        private const int CommandReload = 3;
+
+        private Dictionary<int, string> _injectedJS = new Dictionary<int, string>();
+
+        /// <summary>
+        /// The name of the view manager.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "RCTWebView";
+            }
+        }
+
+        /// <summary>
+        /// The commands map for the webview manager.
+        /// </summary>
+        public override IReadOnlyDictionary<string, object> CommandsMap
+        {
+            get
+            {
+                return new Dictionary<string, object>
+                {
+                    { "goBack", CommandGoBack },
+                    { "goForward", CommandGoForward },
+                    { "reload", CommandReload },
+                };
+            }
+        }
+
+        /// <summary>
+        /// Sets whether a JavaScript is enabled.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
+        [ReactProp("javaScriptEnabled")]
+        public void SetJavaScriptEnabled(Windows.UI.Xaml.Controls.WebView view, bool enabled)
+        {
+            view.Settings.IsJavaScriptEnabled = enabled;
+        }
+
+        /// <summary>
+        /// Sets whether the webpage scales to fit the view and the user can change the scale.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
+        [ReactProp("scalesPageToFit")]
+        public void SetScalesPageToFit(Windows.UI.Xaml.Controls.WebView view, bool enabled)
+        {    
+            // No implementation       
+        }
+
+        /// <summary>
+        /// Used on Android only, controls whether DOM Storage is enabled or not.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
+        [ReactProp("domStorageEnabled")]
+        public void SetDomStorageEnabled(Windows.UI.Xaml.Controls.WebView view, bool enabled)
```

Comment:
Can we create a different prop for the IndexedDb?  I think domStorageEnabled is only specific to Android anyway.